docs/integrations/platforms/kubernetes.mdx

```diff
@@ -67,6 +67,15 @@ spec:
     # The namespace the managed secret should be installed in
     secretNamespace: default
 ```
+<Accordion title="hostAPIReference">
+The `hostAPI` is the host that should be used to pull secrets from and we can use this feature to sync secrets across resources within the same Kubernetes cluster by using the internal DNS name for the Infisical backend service. 
+This is useful for those who want to sync secrets easily and securely within the cluster without exposing them externally.
+	
+  To access the Infisical backend directly you can use a hostAPI configuration as below.
+  ``` bash
+  http://<backend-svc-name>.<namespace>.svc.cluster.local:4000/api
```

Comment:
http worked for me so that's why I suggested it here, also since it's internal I wouldn't have thought it mattered, but lets go with the better option, cheers.

Reply:
https gives me this in operator logs:
Unable to complete api request [err=Get "https://infisical-1677064395-backend.default.svc.cluster.local:4000/api/v2/service-token": http: server gave HTTP response to HTTPS client